lib/core/src/rcMisc.cpp

```diff
@@ -1436,6 +1459,232 @@ clearDataObjCopyInp( void* voidInp ) {
     return;
 }
 
+void clearGeneralAdminInput(void* _p)
+{
+    if (!_p) {
+        return;
+    }
+
+    constexpr auto free_const_pointer = [](const char* _q) {
+        if (_q) {
+            std::free(const_cast<char*>(_q));
+        }
+    };
+
+    auto* q = static_cast<GeneralAdminInput*>(_p);
+
+    free_const_pointer(q->arg0);
+    free_const_pointer(q->arg1);
+    free_const_pointer(q->arg2);
+    free_const_pointer(q->arg3);
+    free_const_pointer(q->arg4);
+    free_const_pointer(q->arg5);
+    free_const_pointer(q->arg6);
+    free_const_pointer(q->arg7);
+    free_const_pointer(q->arg8);
+    free_const_pointer(q->arg9);
+}
```

Comment:
This is actually ok. The data referenced by the pointer doesn't mean the data is actually const. The const-ness can be casted away as demonstrated in the code above.

See https://gcc.godbolt.org/z/Ecxxb3rPj

In case the link dies, the code and compiler explorer options are below.
```cpp
#include <cstdlib>

void no_leaks_or_undefined_behavior()
{
    const auto* p = static_cast<char*>(std::malloc(100));
    auto* q = const_cast<char*>(p);
    std::free(q);
    p = nullptr;
    q = nullptr;
}

int main()
{
    no_leaks_or_undefined_behavior();
}
```
Compiler options: `-std=c++20 -Wall -Wextra -Wpedantic -Wconversion -O0 -g -fsanitize=address,undefined`